src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -38,7 +38,7 @@ const SettingsLMSTab = ({
   const [configsExist, setConfigsExist] = useState(false);
   const [showNoConfigCard, setShowNoConfigCard] = useState(true);
   const [configsLoading, setConfigsLoading] = useState(true);
-  const [displayNames, setDisplayNames] = useState([]);
+  const [displayNames, setDisplayNames] = useState(new Map());
```

Comment:
Is the order of the key-value pairs in displayNames important?

Reply:
No they're not, its just all existing configs and their IDs, it doesn't matter the order just that the info is there

Reply:
Just curious, what is the benefit you were going for in using Map() over a plain {}?

Reply:
My idea was that it's nice that instead of having to loop through every pair to check the associated key which is O(n), it would be nicer to check the value using a map which is O(1)